x/pylons/keeper/msg_server_register_kyc_address.go

```diff
@@ -0,0 +1,44 @@
+package keeper
+
+import (
+	"context"
+
+	"github.com/Pylons-tech/pylons/x/pylons/types"
+	"github.com/cosmos/cosmos-sdk/telemetry"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+func (k msgServer) RegisterKYCAddress(goCtx context.Context, msg *types.MsgRegisterKYCAddress) (*types.MsgRegisterKYCAddressResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	addr, err := sdk.AccAddressFromBech32(msg.Creator)
+	if err != nil {
+		return nil, sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "unable to derive address from bech32 string")
+	}
+
+	acc := k.accountKeeper.GetAccount(ctx, addr)
+	if acc == nil {
+		return nil, sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "account not created")
+	}
+
+	username := types.Username{Value: msg.Username}
```

Comment:
please merge both user name and kyc addr into one message structure

Reply:
Fix in https://github.com/Pylons-tech/pylons/pull/1018/commits/e94ff34e5aebe8a54ee8a8f16ca6ce7a4c4e0b6c